<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="FixedLabel2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2. 调整公式参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixedLabel3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4.选择求解问题</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixedLabel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1. 输入数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>运算开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixedLabel4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>控制台（Console）：</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>当前没有状态</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>186</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>表中读入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>手动输入</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>186</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>电流已知</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>电流未知</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>530</y>
      <width>118</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSelect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件未载入</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OD</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HT</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>c</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Combo：使用逗号区分</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ODEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="HTEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="gEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_13">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>j</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="jEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_15">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_23">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>g</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_24">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VEEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_25">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VE</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_16">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ALEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Iout</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ComboTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ParaDescribe">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>参数说明</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_8">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RinEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rout</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RoutEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RLdsEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_10">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RLds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RhdsEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_11">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rhds</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tdeadEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_17">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ttrEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VinEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VoutEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_18">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Vout</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_19">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ttr</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_20">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>tdead</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_26">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Qg</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VDDEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="QgEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="rcEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_27">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>rc</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_28">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VDD</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_29">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QLabel" name="FixedLabel2_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3. 调整算法参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ParaDescribe2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>参数说明</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_30">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>种群数量（&gt;30）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="popsizeEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="FileStatus_31">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>迭代轮数（&gt;150)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="iterEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
